mike.dessailly/api-limits-page-minor-edit (#13594)
DataDog/documentation: content/en/api/latest/rate-limits/_index.md

All of the API endpoints are rate limited. Once you exceed a certain number of requests in a specific period, Datadog returns an error.

> If you are rate limited, you will see a 429 in the response code. Datadog recommends to either wait the time designated by the `X-RateLimit-Period` before making calls again, or you should switch to making calls at a frequency slightly longer than the `X-RateLimit-Limit` / `X-RateLimit-Period`.

Reviewer: ```suggestion
If you are rate limited, you can see a 429 in the response code. Datadog recommends you either wait the designated time by the `X-RateLimit-Period` before making calls again, or switch to making calls at a frequency slightly longer than the `X-RateLimit-Limit` or `X-RateLimit-Period`.
```
